clients/unshaded/src/main/java/tachyon/client/netty/SingleResponseListener.java

```diff
@@ -26,6 +28,7 @@
 /**
  * A simple listener that waits for a single {@link RPCResponse} message from the remote server.
  */
+@ThreadSafe
```

Comment:
as per our offline discussion, this is technically thread-safe, but I think it makes sense to mark any class with concurrent set and get and not thread safe